cmd/cloud/deployment.go

```diff
@@ -446,7 +446,7 @@ func newDeploymentUpdateCmd(out io.Writer) *cobra.Command {
 	cmd.Flags().StringVarP(&dagDeploy, "dag-deploy", "", "", "Enables DAG-only deploys for the deployment")
 	cmd.Flags().BoolVarP(&cleanOutput, "clean-output", "c", false, "clean output to only include inspect yaml or json file in any situation.")
 	if organization.IsOrgHosted() {
-		cmd.Flags().StringVarP(&schedulerSize, "scheduler-size", "", "", "The size of Scheduler for the Deployment. Possible values can be small, medium, large")
+		cmd.Flags().StringVarP(&schedulerSize, "scheduler-size", "", "", "The size of Scheduler for the Deployment. Possible values can be small, medium, large, extra_large")
```

Comment:
I think it might enforced directly at the Core API level. Can't think of any reason to not add it, wdyt @kushalmalani?

Reply:
The only reason I could think of is that it's for an update operation we can't implement it in the `cmd` package as the deployment type is unknown, and so Core API error might be more meaningful in that sense where it could evaluate the scheduler size option with considering deployment type as well.

Reply:
API side is fine then.